wire-ios/Wire-iOS/Sources/Helpers/String+WireLocales.swift

```diff
@@ -18,14 +18,6 @@
 
 import Foundation
 
-extension String {
-    func capitalizingFirstLetter() -> String {
-        let first = String(prefix(1)).capitalized
```

Comment:
But this code is exactly the same as you added. Why do you want to remove this extension?